### DISCUSSION TITLE
Modification Of Note Template

### DISCUSSION THREAD

**Discussion by prairie-guy** [emacs-citar/citar#721] [Q&A] [answered]
Though a long time emacs user, I am relatively new to `org` and am brand new to `citar`. I am using it to interface with my `bibtex` list of journal articles. I am able to open/create an associated note, but would like to modify the template of a new note. At this point, all I really want to do is change the format of the `title` and include the `doi` and `pmid` fields within my notes. I hope I didn’t miss it in reading the docs or previous discussions. My apologies if I missed it. Any help would be appreciated.

**Comment by bdarcus** [comment, marked as answer]
Hi @prairie-guy  - did you see this section in the README?

https://github.com/emacs-citar/citar#templates

You'd just want to modify that `note` template there.

**Reply by prairie-guy** [reply]
I am a certifiable idiot. I did see it, saw the references to widths within the UI and assumed it was all for the UI. My appreciation and apologies . . .
